executor/testdata/runner/host_fns.py

```
@@ -1,4 +1,5 @@
-from enum import IntEnum
+from enum import IntEnum, StrEnum
```

Ruff (0.11.9)

`enum.StrEnum` imported but unused

Remove unused import: `enum.StrEnum`

(F401)